### PR TITLE
New package: goattracker-2.75

### PR DESCRIPTION
**Pull request by Zeinok**
C64 music tracker

### CODE REVIEW

File: srcpkgs/goattracker/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'goattracker'
+pkgname=goattracker
+version=2.75
+revision=1
+wrksrc=goattrk2/
```
Comment (fosslinux) [inline srcpkgs/goattracker/template:5]: Slash unrequired

File: srcpkgs/goattracker/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+license="GPL-2.0-or-later"
+homepage="https://sourceforge.net/projects/goattracker2/"
+distfiles="${SOURCEFORGE_SITE}/goattracker2/GoatTracker%202/${version}/GoatTracker_${version}.zip"
+checksum=93f6bec01b822093f2fb4f1f220d72d09cba98e5c9c8420bc3666e7ca83da9ce
+
+do_build() {
+	cd ${wrksrc}/src/
+	make clean
+	cd ${wrksrc}/src/bme/
+	make
```
Comment (fosslinux) [inline srcpkgs/goattracker/template:18]: Use `${makejobs}` here as an argument to make for parallelism.

File: srcpkgs/goattracker/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+makedepends="SDL-devel"
+short_desc="C64 music tracker"
+maintainer="Zeinok <zeinok@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://sourceforge.net/projects/goattracker2/"
+distfiles="${SOURCEFORGE_SITE}/goattracker2/GoatTracker%202/${version}/GoatTracker_${version}.zip"
+checksum=93f6bec01b822093f2fb4f1f220d72d09cba98e5c9c8420bc3666e7ca83da9ce
+
+do_build() {
+	cd ${wrksrc}/src/
```
Comment (fosslinux) [inline srcpkgs/goattracker/template:15]: If you want to be fancy, you could go:

```
pushd src
make clean
pushd bme
make
popd
make
popd

**Review by fosslinux — CHANGES_REQUESTED** [review]

**Comment by Zeinok** [comment]
continued in #31074, closing

**Pull request closed without merging**